lms/envs/aws.py

```diff
@@ -1043,3 +1043,36 @@
     ('lsl', 'L'),
     ('gbp', '£'),
 )
+
+
+MPESA_B2C_ACCESS_KEY = AUTH_TOKENS.get('MPESA_B2C_ACCESS_KEY', '')
+MPESA_B2C_CONSUMER_SECRET = AUTH_TOKENS.get('MPESA_B2C_CONSUMER_SECRET', '')
+MPESA_C2B_ACCESS_KEY = AUTH_TOKENS.get('MPESA_C2B_ACCESS_KEY', '')
+MPESA_C2B_CONSUMER_SECRET = AUTH_TOKENS.get('MPESA_C2B_CONSUMER_SECRET', '')
+
+B2C_SECURITY_TOKEN = AUTH_TOKENS.get('B2C_SECURITY_TOKEN', '')
+
+B2C_INITIATOR_NAME = AUTH_TOKENS.get('B2C_INITIATOR_NAME', '')
+B2C_COMMAND_ID = AUTH_TOKENS.get('B2C_COMMAND_ID', '')
+B2C_SHORTCODE = AUTH_TOKENS.get('B2C_SHORTCODE', '')
+B2C_QUEUE_TIMEOUT_URL = AUTH_TOKENS.get('B2C_QUEUE_TIMEOUT_URL', '')
+B2C_RESULT_URL = AUTH_TOKENS.get('B2C_RESULT_URL', '')
+
+C2B_REGISTER_URL = AUTH_TOKENS.get('C2B_REGISTER_URL', '')
+C2B_VALIDATE_URL = AUTH_TOKENS.get('C2B_VALIDATE_URL', '')
+C2B_CONFIRMATION_URL = AUTH_TOKENS.get('C2B_CONFIRMATION_URL', '')
+C2B_SHORT_CODE = AUTH_TOKENS.get('C2B_SHORT_CODE', '')
+C2B_RESPONSE_TYPE = AUTH_TOKENS.get('C2B_RESPONSE_TYPE', '')
+
+C2B_ONLINE_CHECKOUT_URL = AUTH_TOKENS.get('C2B_ONLINE_CHECKOUT_URL', '')
+C2B_ONLINE_CHECKOUT_CALLBACK_URL = AUTH_TOKENS.get('C2B_ONLINE_CHECKOUT_CALLBACK_URL', '')
+C2B_TRANSACTION_TYPE = AUTH_TOKENS.get('C2B_TRANSACTION_TYPE', '')
+C2B_ONLINE_PASSKEY = AUTH_TOKENS.get('C2B_ONLINE_PASSKEY', '')
+C2B_ONLINE_SHORT_CODE = AUTH_TOKENS.get('C2B_ONLINE_SHORT_CODE', '')
+
+# Urls
+GENERATE_TOKEN_URL = AUTH_TOKENS.get('GENERATE_TOKEN_URL', '')
+B2C_URL = AUTH_TOKENS.get('B2C_URL', '')
+
+# number of seconds from the expiry we consider the token expired
+TOKEN_THRESHOLD = ENV_TOKENS.get('TOKEN_THRESHOLD', 600)
```

Comment:
Trailing empty line is missed. Please add it.

Reply:
Added. (https://github.com/raccoongang/edx-platform/pull/536)